src/components/creatEventPageComponents/CreateEvent.jsx

```diff
@@ -0,0 +1,154 @@
+import { addDoc } from "firebase/firestore";
+
+import { eventsCollection } from "@/lib/firebase/controller";
+
+function CreateEvent() {
+    function handleCreateForm(e) {
+        e.preventDefault();
+
+        addDoc(eventsCollection, {
+            title: e.target.title.value,
+            about: e.target.about.value,
+            date: e.target.date.value,
+            image: e.target.image.value,
+            location: e.target.location.value,
+            category: e.target.category.value,
+        }).then(() => {
+            e.target.reset();
+        });
+    }
+
+    return (
+        <div className='bg-white border rounded-lg shadow relative m-10'>
+            <div className='flex items-start justify-between p-5 border-b rounded-t '>
+                <h3 className='text-2xl font-semibold'>Create Event</h3>
+                <button
+                    type='button'
+                    className='text-gray-600 bg-transparent hover:bg-gray-200 hover:text-gray-900 rounded-lg text-sm p-1.5 ml-auto inline-flex items-center'
+                    data-modal-toggle='Event-modal'
+                >
+                    <svg
+                        className='w-5 h-5'
+                        fill='currentColor'
+                        viewBox='0 0 20 20'
+                        xmlns='http://www.w3.org/2000/svg'
+                    >
```

Comment:
i'm not sure what this svg is for ! let me know during today meeting

Reply:
they are extra features (X sign to close window etc..)  of the template provided, in our purpose of usage they can totally be removed